tests/integ/sagemaker/experiments/test_run.py

```diff
@@ -245,6 +246,12 @@ def test_run_from_local_and_train_job_and_all_exp_cfg_match(
                 sagemaker_session=sagemaker_session,
             )
 
+            # the above estimator has wait=True but the job TC could still be receiving updates
+            # after wait is complete resulting in run TC being updated, then when the above with
+            # statement is exited another update trial component call is made _sometimes_
+            # resulting in a ConflictException
+            time.sleep(3)
```

Comment:
If the test captures this ConflictException issue, the same may appear on customer side right?
I saw an internal TT asking for the issue (will send to you offline). Could you help to confirm if it's related?

Reply:
If so, it means customers may already come across the same issue and then should we inform customers to add the sleep time? Or should we add the sleep time somewhere in the service to help unblocking customers?